### ISSUE TITLE
[BUG] Flowise app service deployment error

### ISSUE THREAD

**Issue by benfield97** [FlowiseAI/Flowise#2671]
Status: 500
Error: chatflowsService.getAllChatflows - Error: getRunningExpressApp failed!

Occurs when deploying on Azure via app service. 

btw, the azure docs are really bad and make things very unclear. Improvements to those docs in general would be great.

**Comment by HenryHengZJ** [comment]
PR welcome if you have better suggestion on the docs!

**Comment by ai-Ev1lC0rP** [comment]
From my digging, the issue seems to be related to the MySQL database (or any database) that's hosted in the app service RG. 
The issue resides from the DATABASE_SSL flag in flowise expecting a boolien and the azure app service only accepting an object. 
If i can get it working i'll put the PR in to help mitigate deploying db to Azure issues.

**Comment by nikitas-novatix** [comment]
> From my digging, the issue seems to be related to the MySQL database (or any database) that's hosted in the app service RG. The issue resides from the DATABASE_SSL flag in flowise expecting a boolien and the azure app service only accepting an object. If i can get it working i'll put the PR in to help mitigate deploying db to Azure issues.

Did you manage to get something going @ai-Ev1lC0rP? Any chance you could share your process? Thanks!

**Comment by ai-Ev1lC0rP** [comment]
Yeah i have a PR in . https://github.com/FlowiseAI/Flowise/pull/3364
